policyengine_us/variables/README.md

```diff
@@ -0,0 +1,15 @@
+## Variables
+
+- Useful variable classes:
```

Comment:
I actually dont think we need this, it would be a lot more useful if we just go over general variable structure